ethers-etherscan/src/account.rs

```diff
@@ -63,7 +63,82 @@ mod jsonstring {
     }
 }
 
-/// Possible values for some field responses
+mod json_string {
+    use super::*;
+    use serde::{
+        de::{DeserializeOwned, Error as _},
+        ser::Error as _,
+        Deserializer, Serializer,
+    };
+
+    pub fn serialize<T, S>(value: &Option<T>, serializer: S) -> std::result::Result<S::Ok, S::Error>
+    where
+        T: Serialize,
+        S: Serializer,
+    {
+        let json = match value {
+            Option::None => Cow::from(""),
+            Option::Some(value) => serde_json::to_string(value).map_err(S::Error::custom)?.into(),
+        };
+        serializer.serialize_str(&json)
+    }
+
+    pub fn deserialize<'de, T, D>(deserializer: D) -> std::result::Result<Option<T>, D::Error>
+    where
+        T: DeserializeOwned,
+        D: Deserializer<'de>,
+    {
+        let json = Cow::<'de, str>::deserialize(deserializer)?;
+        if json.is_empty() {
+            Ok(Option::None)
+        } else {
+            let value =
+                serde_json::from_str(&format!("\"{}\"", &json)).map_err(D::Error::custom)?;
```

Comment:
I believe this could be a one-liner by adding `.map(Some)`

Reply:
Done